### PR TITLE
New download progress

### PR DESCRIPTION
**Pull request by LukasHromadnik**
The previous progress UI was not as intutive as we want so we decided to create a new one much nicer progress UI. User can see now the percentage of the download progress.

Also translations were updated.

Simulator Screen Shot - iPhone 11 - 2020-09-01 at 10 05 11

#### Checklist
- [x] Added tests (if applicable)

### CODE REVIEW

File: ACKImagePicker/CircularProgressView.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+            progressLayer?.isHidden = false
+            progressLayer?.strokeEnd = progress
+        }
+    }
+    
+    /// Line width of the circular progress view
+    public var lineWidth: CGFloat = 2
+    
+    /// Color applied on the background circle
+    public var secondaryTintColor = UIColor(white: 230/255, alpha: 1)
```
Comment (olejnjak) [inline ACKImagePicker/CircularProgressView.swift:30]: ❓ black color?

**Review by olejnjak — COMMENTED** [review]
Maybe it would also be nice to run `swiftlint autocorrect` so those warnings disappear.

File: ACKImagePicker/CircularProgressView.swift [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import UIKit
+
+/// Progress view that is in the shape of a circle
+///
+/// It draws itself within specified rect.
+/// The size is derived from the size of the rect where the smaller dimension (width / height) is used.
+/// The view itslef is drawn in the middle of the rect.
```
Comment (fortmarek) [inline ACKImagePicker/CircularProgressView.swift:7]: typo in `itself`

**Review by fortmarek — APPROVED** [review]

File: ACKImagePicker/CircularProgressView.swift (hunk 2 above)
Comment (LukasHromadnik) [reply ACKImagePicker/CircularProgressView.swift]: fixed ✅

**Comment by LukasHromadnik** [comment]
`swiftlint autorrect` ✅

**Comment by LukasHromadnik** [comment]
waiting for @janmisar's feedback

**Review by janmisar — APPROVED** [review]

**Review by olejnjak — APPROVED** [review]

**Pull request merged**